### ISSUE TITLE
Train-Test Split

### ISSUE THREAD

**Issue by ARUNAGIRINATHAN-K** [ARUNAGIRINATHAN-K/ARUNAGIRINATHAN-K#1]
from sklearn.model_selection import train_test_split

X = df.drop('Close', axis=1)
y = df['Close']
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, shuffle=False)
